crates/bevy_picking/src/events.rs

```diff
@@ -750,6 +765,28 @@ pub fn pointer_events(
                     event_writers.move_events.send(move_event);
                 }
             }
+            PointerAction::Scroll { x, y, unit } => {
+                // If it's a press, emit a Pressed event and mark the hovered entities as pressed
```

Comment:
this comment looks like a copy paste error

Reply:
I removed that and added an appropriate comment mirroring the other events' implementations lower down in that code block.